stix_shifter/scripts/stix_shifter.py

```diff
@@ -150,29 +151,29 @@ def main():
 
     help_and_exit = args.command is None
 
-    if 'module' in args:
-        args_module_dialects = args.module
-        
-        options = None
-        if 'options' in args:
-            options = args.options
-        if options == None:
-            options = {}
-        else:
-            options = json.loads(options)
-
-        module, dialects = process_dialects(args_module_dialects, options)
-        args.options = json.dumps(options)
-
-        all_modules = module_list()
-        if not module in all_modules:
-            print(f"module '{module}' is not found")
-            help_and_exit = True
+  if 'module' in args:
```

Comment:
@yurii-klymenko I get an indentation error in this line. if you see the same error-
`Traceback (most recent call last):
  File "main.py", line 1, in <module>
    from stix_shifter.scripts.stix_shifter import main
  File "/Users/mdazam/uds_workspace/connector/stix-shifter/stix_shifter/scripts/stix_shifter.py", line 154
    if 'module' in args:
                       ^
IndentationError: unindent does not match any outer indentation level`

you may need to properly tab this line.

Reply:
it is fixed by Danny in v3-master